cli/reference/cmdline/script/script-run.md

```diff
@@ -52,6 +52,12 @@ Run a script called "deploy" on all changed stacks where it is available:
 terramate script run --changed deploy
 ```
 
+Run a script called "deploy" on all changed stacks (not considering uncommitted files):
```

Comment:
The `script run` does not follow the doc style of other commands where we document each individual flag. Here are only examples..

Reply:
will add a follow up task for that